install.sh

```diff
@@ -153,7 +172,11 @@ dep_get ()
     if ! sha256_verify "${pkg_hash}" "${pkg_name}"; then
         return 1
     fi
-    tar -xzf "${pkg_name}" -C ../
+    if [[ $1 == *zip ]]; then
+        unzip "${pkg_name}" -d ../
```

Comment:
as mentioned: https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000#issuecomment-962028424 adding `-o` gets rid of the overwrite confirmation question with unzip.
Tested ok with:
` unzip "${pkg_name}" -od ../`

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000/commits/7a5e4c3369966dda121ac01a09dd999071c8dc4a